engine/schema/src/com/cloud/upgrade/dao/Upgrade452to460.java

```diff
@@ -60,10 +60,25 @@ public boolean supportsRollingUpgrade() {
 
     @Override
     public void performDataMigration(final Connection conn) {
+        alterAddColumnToCloudUsage(conn);
         updateVMInstanceUserId(conn);
         addIndexForVMInstance(conn);
     }
 
+    public void alterAddColumnToCloudUsage(final Connection conn) {
+        final String alterTableSql = "ALTER TABLE `cloud_usage`.`cloud_usage` ADD COLUMN `quota_calculated` tinyint(1) DEFAULT 0 NOT NULL COMMENT 'quota calculation status'";
+        try (PreparedStatement pstmt = conn.prepareStatement(alterTableSql)) {
+            pstmt.executeUpdate();
+            s_logger.debug("Altered usage table and added column quota_calculated");
+        } catch (SQLException e) {
+            if (e.getMessage().contains("quota_calculated")) {
+                s_logger.debug("Usage table already has a column called quota_calculated");
```

Comment:
Why don't we log this message to ``INFO`` or ``WARN``?  Thinking through it as a sys admin, I would like to know when a migration decided not to perform a migration -- help me understand the impact of partial upgrade.